### ISSUE TITLE
Player does not hide #ima-ad-container and does not fire adserror on VPAID error

### ISSUE THREAD

**Issue by avdeev** [googleads/videojs-ima#243]
VAST: https://data.videonow.ru/?profile_id=899365&format=vast

I tested this VAST on Advanced example and got follows errors

```
videojs.ima.js:726 WARNING: player.ima.setContent is deprecated. Use player.ima.setContentWithAdTag instead.

videojs.ima.js:412 Uncaught TypeError: Cannot read property 'isLinear' of undefinedplayer.ima.onAdComplete_ @ videojs.ima.js:412Hf @ ima3.js:74h.dispatchEvent @ ima3.js:72Y.A @ ima3.js:289Y.oa @ ima3.js:285Hf @ ima3.js:74h.dispatchEvent @ ima3.js:72Zm.o @ ima3.js:258Ef @ ima3.js:69Af @ ima3.js:70(anonymous function) @ ima3.js:68
```

Reproduced #170

**Comment by shawnbuso** [comment]
Have you tested with the latest version of the plugin? I tested with our sample at http://googleads.github.io/videojs-ima/examples/advanced/ and I'm able to use the controls after the ad errors and I see the error in the dev console.

**Comment by mkhazov** [comment]
@shawnbuso 
VMAP Post-roll: https://pubads.g.doubleclick.net/gampad/ads?sz=640x480&iu=/124319096/external/ad_rule_samples&ciu_szs=300x250&ad_rule=1&impl=s&gdfp_req=1&env=vp&output=vmap&unviewed_position_start=1&cust_params=deployment%3Ddevsite%26sample_ar%3Dpostonly&cmsid=496&vid=short_onecue&correlator=

#ima-ad-container is shown over the player and blocks player controls.
Applying "display: none" instead of  "display: block" to #ima-ad-container fixes the problem.

Tested on https://googleads.github.io/videojs-ima/examples/advanced/

**Comment by shawnbuso** [comment]
Thanks for the info - this will be fixed by #245
